BuildResidentialHPXML/measure.rb

```diff
@@ -3152,6 +3158,11 @@ def argument_errors(args)
     error = (args[:window_aspect_ratio] <= 0)
     errors << 'Window aspect ratio must be greater than zero.' if error
 
+    if args[:site_time_zone_utc_offset].is_initialized
+      error = (args[:site_time_zone_utc_offset].get < -12) || (args[:site_time_zone_utc_offset].get > 14)
+      errors << 'Time zone UTC offset must be between -12 and 14.' if error
+    end
```

Comment:
It seems like we can remove this and rely on the downstream error-checking in EPvalidator.xml, right?